media/selenium/scripts/htmlutils.js

```diff
@@ -752,7 +752,7 @@ Assert.equals = function() {
                 "' but was '" + args.actual + "'");
 };
 
-Assert.assertEquals = Assert.equals;
+Assert.assertEqual = Assert.equals;
```

Comment:
I'm not exactly sure what this selenium JS script is doing, but I think this line should be kept as is. I don't think this is related to the Django `assertEqual`.